compiler/ballerina-tools-api/src/main/java/io/ballerina/tools/text/LineRange.java

```diff
@@ -25,22 +25,27 @@
  * @since 2.0.0
  */
 public class LineRange {
-    private final String filePath;
+    private final String fileName;
     private final LinePosition startLine;
     private final LinePosition endLine;
 
-    private LineRange(String filePath, LinePosition startLine, LinePosition endLine) {
-        this.filePath = filePath;
+    private LineRange(String fileName, LinePosition startLine, LinePosition endLine) {
+        this.fileName = fileName;
         this.startLine = startLine;
         this.endLine = endLine;
     }
 
-    public static LineRange from(String filePath, LinePosition startLine, LinePosition endLine) {
-        return new LineRange(filePath, startLine, endLine);
+    public static LineRange from(String fileName, LinePosition startLine, LinePosition endLine) {
+        return new LineRange(fileName, startLine, endLine);
     }
 
+    @Deprecated
```

Comment:
Why don't we have documentation for these methods? Could've suggested `fileName` instead too.